app/src/main/java/com/github/sdp/mediato/data/Utils.java

```diff
@@ -0,0 +1,78 @@
+package com.github.sdp.mediato.data;
+
+import com.github.sdp.mediato.model.Location;
+import com.github.sdp.mediato.model.User;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.ValueEventListener;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * This is a Utils class for the database classes
+ */
+class Utils {

```

Comment:
I think changing the name to `DatabaseUtils` or something would be better in case there are other utils classes in the future.